WordPress/src/main/res/values/strings.xml

```diff
@@ -1095,7 +1095,7 @@
     <string name="scan_idle_manual_scan_description">To review your site again run a manual scan, or wait for Jetpack to scan your site later today.</string>
     <string name="scan_idle_last_scan_description">The last Jetpack scan ran %s and did not find any risks. %s</string>
     <string name="scan_idle_with_threats_description_plural">The scan found %1s potential threats with %2s. Please review them below and take action or tap the fix all button. We are %3$s if you need us.</string>
-    <string name="scan_idle_with_threats_description_singular">The scan found %1s potential threat with %2s. Please review the threat below and take action or tap the fix all button. We are %3$s if you need us.</string>
+    <string name="scan_idle_with_threats_description_singular">The scan found 1 potential threat with %1s. Please review the threat below and take action or tap the fix all button. We are %2$s if you need us.</string>
```

Comment:
Wording nitpick: I know that the previous code was written in a way that the output was already using `1` (and not `one`) in singular form (since it used via `UiStringText("${fixableThreatIds.size}")` in both code paths), but if I'm not mistaken, at least in English it's better to spell-out numbers when using in that kind of context ([source](https://www.grammarly.com/blog/when-to-spell-out-numbers/)) 🙂 

```suggestion
    <string name="scan_idle_with_threats_description_singular">The scan found one potential threat with %1s. Please review the threat below and take action or tap the fix all button. We are %2$s if you need us.</string>
```
(ditto for the other string below of course)

Reply:
I thought about it but saw that using "1" was a common pattern, I'll leave the decision on this up to @ashiagr though 🙂

Reply:
Actually yes, 1 is a common pattern in the code but I think it is fine to use "one" as well if it eases our translators job. I'm fine with replacing "1" with "one".